odonto/models.py

```diff
@@ -129,6 +129,43 @@ class Meta:
         verbose_name_plural = "Patient information"
 
 
+class CovidStatus(models.EpisodeSubrecord):
+    """
+    This model appears on both FP17 and FP17O forms
+    and tracks the number of calls of patients of
+    various covid status.
+
+    A patient can encompass different stages
```

Comment:
I think the word encompass is doing a lot of work here ?
Can we say this thought clearer?